### PR TITLE
Expose reading attempts in Plexon2

### PR DESCRIPTION
**Pull request by h-mayorquin**
We have been working on this neo, but I forgot to expose it here in Spikeinterface. It would be great if we could expose it here before the next patch release.

### CODE REVIEW

**Review by zm711 — COMMENTED** [review]
One typo and one comment @h-mayorquin :)

File: src/spikeinterface/extractors/neoextractors/plexon2.py [hunk 1]
```diff
@@ -28,6 +28,10 @@ class Plexon2RecordingExtractor(NeoBaseRecordingExtractor):
             ids: ["source3.1" , "source3.2", "source3.3", "source3.4"]
     all_annotations : bool, default: False
         Load exhaustively all annotations from neo.
+    readding_attemps : int, default: 25
+        Number of attempts to read the file before raising an error
+        This opening process is somewhat unreliable and might fail occasionally. Adjust this higher
```
[the remark this answers is not in the document]
Comment (h-mayorquin) [reply src/spikeinterface/extractors/neoextractors/plexon2.py:33]: You think that would be helpful? if so, yes, by all means, can you make a suggestion?

**Comment by zm711** [comment]
Weird test failure? Was the wrong info being bubbled up? Although it worked on Windows....

**Comment by h-mayorquin** [comment]
This is an environment issue. An older version of neo is being installed for some reason.

**Comment by alejoe91** [comment]
@h-mayorquin the stream name in the test seem to be triggering an error. Can you check?

**Comment by h-mayorquin** [comment]
> @h-mayorquin the stream name in the test seems to be triggering an error. Can you check?

It is what I mentioned to Zach I think Something in our CI is not installing neo dev! : O

I will check it.

**Comment by zm711** [comment]
We were missing the after-release. Fixed here #3440.

**Comment by h-mayorquin** [comment]
Thanks guys @zm711 @alejoe91 that fixed it.

**Comment by h-mayorquin** [comment]
This is ready.

**Review by samuelgarcia — APPROVED** [review]

**Pull request merged**